jenkins/jobs/bmo_e2e_tests.pipeline

```diff
@@ -7,7 +7,17 @@ def TIMEOUT = 3600
 def branch = env.ghprbActualCommit ?: "main"
 def repoUrl = env.ghprbAuthorRepoGitUrl ?: "https://github.com/metal3-io/baremetal-operator.git"
 
+// Define GINKGO_FOCUS var based on the test set
+def GINKGO_FOCUS = ""
+
+if (TEST_SET == "e2e-upgrade-test") {
+  GINKGO_FOCUS="upgrade"
+}
```

Comment:
Why not have GINKGO_FOCUS directly as a parameter?

Reply:
I'm not sure if jjb allows that :-?

Reply:
Please check my reply in https://gerrit.nordix.org/c/infra/cicd/+/20387